### ISSUE TITLE
verses to bible text

### ISSUE THREAD

**Issue by fenimi** [Niger-Volta-LTI/yoruba-text#14]
Is it possible to add the verse numbers to the bible text?

**Comment by ruohoruotsi** [comment]
Yes it is, I have a have updated the scraper to do so on a per-verse basis. Give me a few days to prepare it for public consumption and re-scrape. 

@fenimi I'm curious about your application or per-verse usage. Cheers.

**Comment by ruohoruotsi** [comment]
<img width="708" alt="Screen Shot 2019-06-04 at 11 52 31 PM" src="https://user-images.githubusercontent.com/2321161/58936556-deea9a00-8724-11e9-9daa-20cb66e9d1b5.png">
How about this? Screenshot from 1st Chronicles chapter 4, verse 2 (https://www.bible.com/bible/911/1CH.4.YCB)

```
2 Reaiah ọmọ Ṣobali ni baba Jahati, àti Jahati baba Ahumai àti Lahadi. 
Àwọn wọ̀nyí ni ẹ̀yà ará Sorati.
```

**Comment by fenimi** [comment]
Hi, Thank you so much for replying. I will be looking forward to seeing it. I'm a linguist trying to gather as much Yoruba data for research especially in their original state. 

I'll be looking forward to the update.
Regards,Ife    On Wednesday, June 5, 2019, 1:08:56 AM GMT+3, Ruoho Ruotsi <notifications@github.com> wrote:  
 
 
Yes it is, I have a have updated the scraper to do so on a per-verse basis. Give me a few days to prepare it for public consumption and re-scrape.

@fenimi I'm curious about your application or per-verse usage. Cheers.

**Comment by fenimi** [comment]
Hi,Yes, this looks good.
Thanks a lot.
Regards,Ife
    On Wednesday, June 5, 2019, 10:07:09 AM GMT+3, Ruoho Ruotsi <notifications@github.com> wrote:  
 
 

How about this? Screenshot from 1st Chronicles chapter 4, verse 2 (https://www.bible.com/bible/911/1CH.4.YCB)2 Reaiah ọmọ Ṣobali ni baba Jahati, àti Jahati baba Ahumai àti Lahadi. 
Àwọn wọ̀nyí ni ẹ̀yà ará Sorati.

**Comment by ruohoruotsi** [comment]
Fixed in https://github.com/Niger-Volta-LTI/yoruba-text/commit/bf5fb228da7a71b14767710f0e21cc5164da8a10

@fenimi Can you please check that the update satisfies your request. Note that I added two separate versions the BSN and Biblica versions. Cheers.

**Comment by fenimi** [comment]
Thank you so much. This looks good thanks a lot.
Regards,Ife
    On Wednesday, June 26, 2019, 11:38:15 PM PDT, Ruoho Ruotsi <notifications@github.com> wrote:  
 
 
Fixed in bf5fb22

@fenimi Can you please check that the update satisfies your request. Note that I added two separate versions the BSN and Biblica versions. Cheers.
